app/assets/stylesheets/style/not_used/testimonials.scss

```diff
@@ -0,0 +1,133 @@
+// Not used and too long dwessman
+.testimonials {
+  margin-bottom: 10;
+}
+
+.testimonials .testimonial-info {
+  color: $fsek-orange;
+  font-size: 16px;
+  margin-top: 18px;
+  padding: 0 15px;
+}
+
+.testimonials .testimonial-info span {
+  position: relative;
+  top: 3px;
+}
+
+.testimonials .testimonial-info em {
+  color: $gray-text;
+  display: block;
+  font-size: 13px;
+}
+
+.testimonials .testimonial-info img {
+  border-radius: 50% !important;
+  border: solid 1px #ccc;
+  float: left;
+  height: 60;
+  margin-right: 15px;
+  padding: 2px;
+  width: 60;
+}
+
+.testimonials .testimonial-author {
+  overflow: hidden;
+}
+
+.testimonials .carousel-arrow {
+  top: -65px;
+  position: relative;
+}
+
+.testimonials .carousel-arrow i {
+  background: #fafafa;
+  color: $gray-text;
+  font-size: 20;
+  padding: 2px 8px;
+}
+
+.testimonials .carousel-arrow i:hover {
+  background: $fsek-orange;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.